addons/skin.estuary/xml/Includes_DialogSelect.xml

```diff
@@ -349,4 +503,106 @@
 			</control>
 		</control>
 	</include>
+	<include name="GameDialogSelectSaveInGameLayout">
+		<control type="button">
+			<description>background close area</description>
+			<left>0</left>
+			<top>0</top>
+			<width>100%</width>
+			<bottom>540</bottom>
+			<texturefocus />
+			<texturenofocus />
+			<onclick>Action(close)</onclick>
+		</control>
+		<control type="group">
+			<bottom>0</bottom>
+			<height>540</height>
+			<width>100%</width>
+			<control type="image">
+				<animation effect="fade" start="0" end="100" time="300">WindowOpen</animation>
+				<animation effect="fade" start="100" end="0" time="300">WindowClose</animation>
+				<texture colordiffuse="E6FFFFFF">dialogs/dialog-bg-nobo.png</texture>
+			</control>
+			<control type="panel" id="11">
+				<top>30</top>
+				<scrolltime tween="sine">200</scrolltime>
+				<orientation>horizontal</orientation>
+				<itemlayout width="480" height="340">
+					<control type="group">
+						<left>18</left>
+						<right>18</right>
+						<top>5</top>
+						<control type="image">
+							<width>444</width>
+							<height>250</height>
+							<aspectratio>scale</aspectratio>
+							<texture border="4">$VAR[InfoWallThumbVar]</texture>
```

Comment:
Here you use `$VAR[InfoWallThumbVar]`.

The variable [resolves](https://github.com/garbear/xbmc/blob/retroplayer-18.7/addons/skin.estuary/xml/Variables.xml#L70) to:

* `$INFO[Listitem.Art(poster)]` if there is fanart
* `$INFO[ListItem.Icon]` if there is not

because we don't expect to use fanart for icons, let's set this value to `$INFO[ListItem.Icon]` directly.

Reply:
same thing here https://github.com/NikosSiak/xbmc/blob/savestate-manager/addons/skin.estuary/xml/Includes_DialogSelect.xml#L242